lib/internal/child_process.js

```diff
@@ -699,6 +702,20 @@ function setupChannel(target, channel, serializationMode) {
     });
   });
 
+  target.on('newListener', function() {
+    if (!target.channel) return;
+
+    process.nextTick(() => {
+      const messages = target.channel[kPendingMessages];
+      if (!messages.size || !target.listenerCount('message')) return;
```

Comment:
As the CI error shows, `target.channel` may have been deleted by the time the next tick runs.
```suggestion
      if(target.listenerCount('message') === 0 || target.channel === null) return;
      const messages = target.channel[kPendingMessages];
      if (!messages.size) return;
```

Reply:
uuh, perfect! just fixed!